.github/workflows/self-push.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           OMP_NUM_THREADS: 8
           MKL_NUM_THREADS: 8
           HF_HOME: /mnt/cache
+          TRANSFORMERS_IS_CI: yes
```

Comment:
You can set this at the top level to specify it only once (cf https://github.com/huggingface/tokenizers/blob/master/.github/workflows/python-release.yml#L8)

Reply:
maybe we can also do that for Circle CI for consistency?